### ISSUE TITLE
on window resize

### ISSUE THREAD

**Issue by doespresso** [williamngan/pt#28]
Please, help
How to make canvas fluid, or simply reinit when window resizing.
Applying to the demos - I want make something interesting on my website with them.

**Comment by williamngan** [comment]
@doespresso Yep, there's a callback function to listen to resize event. Take a look:

Demo: http://williamngan.github.io/pt/demo/index.html?name=canvasspace.resize
Demo source code: https://github.com/williamngan/pt/blob/master/demo/canvasspace.resize.js#L48
Doc: http://williamngan.github.io/pt/docs/#func-CanvasSpace-resize

Hope this helps!
